http/rest-client-reactive/src/main/java/io/quarkus/ts/http/restclient/reactive/resources/HttpVersionBaseResource.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.ts.http.restclient.reactive.resources;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.QueryParam;
+import jakarta.ws.rs.core.Response;
+
+import io.smallrye.mutiny.Uni;
+
+@ApplicationScoped
+@Path("/httpVersion")
```

Comment:
```suggestion
@Path("/http-version")
```
That's usual convention, or are you testing whether it works with case sensitive path? If so, let's document.